### ISSUE TITLE
Unit tests will never work on our CI

### ISSUE THREAD

**Issue by JagCesar** [JagCesar/Tesla-API#24]
Our unit tests require valid credentials to be successful. How can we work around that?

Mocked data objects maybe? But then we have to automate the creation of those objects if Tesla desires to update the JSON object returned for any endpoint. 🤔

**Comment by westerlund** [comment]
But at that point, they'll have to release a new version of their app! I think you're perfectly safe with mocking all the endpoints. You can just anonymize previous requests or just have the fields that you care about :)

**Comment by JagCesar** [comment]
@westerlund That's true. Let's start out that way, and then see where it takes us :)
